### PR TITLE
Edits to rate limiting task

### PR DESCRIPTION
**Pull request by gargnupur**
Some minor nits to rate limiting task

### CODE REVIEW

**Comment by tangiel** [comment]
@googlebot rescan

**Comment by sdake** [comment]
Note the gate is failing: ./content/docs/tasks/policy-enforcement/denial-and-list/index.md:176: MD009 Trailing spaces
```
./content/docs/tasks/policy-enforcement/denial-and-list/index.md:178: MD009 Trailing spaces
./content/docs/tasks/policy-enforcement/denial-and-list/index.md:170: MD022 Headers should be surrounded by blank lines
./content/docs/tasks/policy-enforcement/denial-and-list/index.md:175: MD031 Fenced code blocks should be surrounded by blank lines
./content/docs/tasks/policy-enforcement/denial-and-list/index.md:179: MD031 Fenced code blocks should be surrounded by blank lines
```

File: content/docs/tasks/policy-enforcement/denial-and-list/index.md [hunk 1]
```diff
@@ -168,6 +168,17 @@ subnet.
     not whitelisted`
 
 ## Cleanup
+* Remove the Mixer configuration for simple denials:
+
+    {{< text bash >}}
+    $ kubectl delete -f @samples/bookinfo/policy/mixer-rule-deny-label.yaml@
+    {{< /text >}}
+    
+    If you are using Istio 1.1.2 or prior:
```
Comment (sdake) [inline content/docs/tasks/policy-enforcement/denial-and-list/index.md]: I think this is against docs review policy (@geeknoid correct me if I'm wrong please).  I think we want to take the position that when operators read the documentation they are accessing documentation for the specific release they are reviewing.  As this is a master PR (which will seed release-1.2 branch when @geeknoid branches the repo), 1.1.2 is not relevant.  If you want 1.1.2 commentary, please submit a PR to release-1.1 and operators will be able to access the 1.1.2 documentation directly.

Comment (gargnupur) [reply content/docs/tasks/policy-enforcement/denial-and-list/index.md]: Removed 1.1.2 or prior comment..

File: content/docs/tasks/policy-enforcement/control-headers/index.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ This task demonstrates how to use a policy adapter to manipulate request headers
   configuration for the `httpbin` service containing two route rules that allow traffic for paths `/headers` and
   `/status`:
 
-    {{< text yaml >}}
+    {{< text bash >}}
```
Comment (geeknoid) [inline content/docs/tasks/policy-enforcement/control-headers/index.md]: Actually, this should be "text bash yaml" indicating a command line, followed by yaml output.

**Pull request merged**